pkgs/applications/misc/flashprint/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, glibc
+, gcc-unwrapped
+, autoPatchelfHook
+, libGLU
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation {
+  name = "flashprint";
+  version = "5.6.1";
+  src = fetchurl {
+    url = "https://en.fss.flashforge.com/10000/software/01e7742d5a721201cd33d4f8c083b63b.deb";
+    sha256 = "sha256-lvK4igv/L1zrqNtqbWtCyZrSZbkY8Ak/Li3NeVAH1YU=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook dpkg wrapQtAppsHook ];
+  buildInputs = [ glibc gcc-unwrapped libGLU qtbase ];
```

Comment:
The only one of these actually needed is `libGLU`, as the others are either contained in `stdenv` or `wrapQtAppsHook`